docs/objects/router.md

```diff
@@ -126,11 +126,13 @@ Each routing method described above accepts a callback routine as its final argu
 **Arguments**
 : The third argument is an associative array that contains values for the current route's named placeholders.
 
+### Writing content to the response
+
 There are two ways you can write content to the HTTP response. First, you can simply `echo()` content from the route callback. This content will be appended to the current HTTP response object. Second, you can return a `Psr\Http\Message\ResponseInterface` object.
 
 ### Closure binding
 
-If you use a `Closure` instance as the route callback, the closure's state is bound to the `\Slim\App` instance. This means you can access the `\Slim\App` object from inside the route callback with `$this`. Becaues the `\Slim\App` is itself an instance of `\Pimple\Container`, you can quickly access any registered Pimple services from inside the Closure callback like this:
+If you use a `Closure` instance as the route callback, the closure's state is bound to the DI container. This means you can access the DI container from inside the route callback with `$this`. This allows you to quickly access any services registered in the DI container from inside the Closure callback like this:
```

Comment:
Ah, no problem — I was sure I saw something bound to the container and that explains it. I'll revert this part...